website/docs/reference/resource-configs/singlestore-configs.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "SingleStore configurations"
+id: "singlestore-configs"
+---
+
+
+<Changelog>
+
+  - **v1.1.1:** Added support for for `storage_type`, `indexes`, `primary_key`, `sort_key`, `shard_key`, `unique_table_key`, `charset`, `collation`  options for creating SingleStore tables.
+
+</Changelog>
+
+## Performance Optimizations
+[SingleStore Physical Database Schema Design documentation](https://docs.singlestore.com/managed-service/en/create-a-database/physical-database-schema-design/concepts-of-physical-database-schema-design.html) is helpful if you want to use specific options (that are described below) in your dbt project.
+
+
+### Storage type
+SingleStore supports two storage types: **In-Memory Rowstore** and **Disk-based Columnstore** (the latter is default). See [the docs](https://docs.singlestore.com/managed-service/en/create-a-database/physical-database-schema-design/concepts-of-physical-database-schema-design/choosing-a-table-storage-type.html) for details. The dbt-singlestore adapter allows you to specify which storage type your table materialization would rely on using `storage_type` config parameter. 
+
+<File name='rowstore_model.sql'>
+
+```sql
+{{ config(materialized='table', storage_type='rowstore') }}
+
+select ...
+```
+
+</File>
+
+### Keys
+
```

Comment:
any reason you're using a comma-separated string instead of an actual list object?

Reply:
Decided to use a list for consistency with `indexes` option